src/core/ui/actions/SaveButton.tsx

```
@@ -1,19 +1,18 @@
 import React from 'react';
-import { LoadingButton, LoadingButtonProps } from '@mui/lab';
-import { ButtonTypeMap } from '@mui/material/Button/Button';
+import { Button, ButtonProps, ButtonTypeMap } from '@mui/material';
 import { useTranslation } from 'react-i18next';
 import SaveIcon from '@mui/icons-material/Save';
 
 const SaveButton = <D extends React.ElementType = ButtonTypeMap['defaultComponent'], P = {}>({
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)